package.json

```diff
@@ -81,6 +81,7 @@
     "devDependencies": {
         "babel-eslint": "8.2.6",
         "babel-jest": "24.9.0",
+        "babel-plugin-polyfill-corejs2": "0.2.2",
```

Comment:
is this needed?

Reply:
In a way. Without this all unit tests fail as this module is not found. This is not at all related to this PR, but it allows me to commit without needing `--no-verify` flag